### ISSUE TITLE
Scala JS source maps, map to release dir?

### ISSUE THREAD

**Issue by Quafadas** [outr/scribe#378]
In my pet project, vite complains about source map errors. I believe one of the culprits to be scribe.

Here's a sample .js.map file.
```
{
"version": 3,
"file": "frontend.exposed.historicEventSetRender$.js",
"mappings": "blah",
"sources": ["file:///C:/temp/ilsclaims/frontend/src/Pages/historicEventSetRender.scala", "https://raw.githubusercontent.com/raquo/Airstream/c769f6591e7f6f1377154d484ce8657b752aa179/src/main/scala/com/raquo/airstream/state/SourceVar.scala","file:///home/mhicks/projects/open/scribe/core/shared/src/main/scala/scribe/LoggerSupport.scala"],
"names": ["thisEvent", "evSignal", "allEvents", "dateUpdater", "perilUpdater", "historicCatUpdater", "commentaryUpdater", "shareDriveUpdater", "eventUpdater", "internal-b28b7af69320201d1cf206ebf28373980add1451", "this", "data", "newValue", "internal-cc9ed62c4da3ee05f089bcd2b6206c66f470149f", "arg1", "internal-886c06f6e24163ae65bd354a00fd4306a5b3dba5", "mod", "item", "temp", "s", "internal-e695ddc18dc4fc6b9c0c016cece5a3161cf42db4", "histEventSetId", "eventId", "asOfDate", "commentary", "sourceFolder", "sseids", "category", "modelVersion", "attrWrapperDiv", "internal-a08ca37f31725b66f4cce58b26ae7226a5925624", "apply", "p", "frontend.Page$SingleHistoricEvent", "source", "internal-f828b5c57d60e24e5681c9358eabe9187c5269ee", "_$2", "internal-f77845442c276475d33e90006f741349bb7a6994", "separator", "_$3", "_$5", "mods", "internal-45b4f1c1e40f2066d87319099cb5cb2aea6c42c0", "project", "x$1", "_$6", "_$9", "internal-1c753a45cee36080dea899db4db07e2b857fc77a", "_$12", "_$15", "_$18", "histEv", "sse", "internal-fdec4afa56fd2d81f5fcbd2a9a40c388dafb21c2"],
"lineCount": 492
}
```
I think the important part, is the file:///home/mhicks/projects/open/scribe/core/shared/src/main/scala/scribe/LoggerSupport.scala line, which obviously doesn't exist, in my local environment. Because I lurk discord, I believe the solution may look something like this;

https://discord.com/channels/632150470000902164/635668814956068864/1060561961810677760

https://github.com/typelevel/sbt-typelevel/blob/4da726ca021ea2c6459e8286d9ec49ab7c79a883/github/src/main/scala/org/typelevel/sbt/TypelevelScalaJSGitHubPlugin.scala#L35

I believe there to be a compiler flag, which could potentially point this back to this repository, for example. I will openly admit, that my understanding of this, is somewhat superficial...

**Comment by darkfrog26** [comment]
Great find! I'll try to find some time to dig into this and figure out how to properly fix it. If you stumble upon a solution before I do, please feel free to submit a PR and I'll get a new release out ASAP.

**Comment by Quafadas** [comment]
I've filed a bunch of these; I know typelevel have it solved by sbt-typelevel which was linked above. Laminar also appears to have a good solution. Which I _think_ is this line; 
https://github.com/raquo/Laminar/blob/739cc0dbab319c685f28be30a1592b7d04347842/build.sbt#L76

https://github.com/disneystreaming/smithy4s/issues/705

**Comment by Quafadas** [comment]
Huh... it's already here; 
https://github.com/outr/scribe/blob/d145d24fc96002ffc6e09024f1d88ab5be3851c8/build.sbt#L84

**Comment by darkfrog26** [comment]
What version of Scribe are you using?

**Comment by Quafadas** [comment]
val scribeVersion   = "3.10.5"

Which I think is pretty recent.

**Comment by Quafadas** [comment]
Aha! 

I think the flag you have set, would work for scala 2, but not scala 3... 
https://github.com/typelevel/sbt-typelevel/blob/4da726ca021ea2c6459e8286d9ec49ab7c79a883/github/src/main/scala/org/typelevel/sbt/TypelevelScalaJSGitHubPlugin.scala#L35

**Comment by darkfrog26** [comment]
@Quafadas awesome work! I'll take a look now.

**Comment by darkfrog26** [comment]
Just released `3.10.6` which should resolve this. Let me know if you still have any trouble, and thanks for providing the fix!
